app/controllers/obs_controller.rb

```diff
@@ -65,7 +65,7 @@ def filter_packages
     @packages.reject! { |p| p.project.start_with?("openSUSE:Factory:Staging") }
 
     # only show packages
-    @packages = @packages.reject { |p| p.first.type == 'ymp' }
+    @packages.reject! { |p| p.type == 'ymp' }
```

Comment:
This is not really related to what the commit message says, right?

Reply:
This line needs to be edited to remove the `.first`. Would you like me to move all the `foo = foo.reject` -> `foo.reject!` (respectively `.select`) changes to another commit?

Reply:
It is ok, up to you! :wink: